platform/security/idp/security-idp-plugin/security-idp-plugin-namequalifier/src/test/groovy/org/codice/ddf/security/idp/plugin/namequalifier/NameQualifierPresignPluginSpec.groovy

```diff
@@ -93,12 +92,33 @@ class NameQualifierPresignPluginSpec extends Specification {
 
         then:
         1 * nameID.getFormat()
```

Comment:
❓ Shouldn't this be `2 *`? I would think `getFormat()` would be called twice here: https://github.com/bantillo/ddf/blob/05f34805478cf1061664f142b29a2a25645fec35/platform/security/idp/security-idp-plugin/security-idp-plugin-namequalifier/src/main/java/org/codice/ddf/security/idp/plugin/namequalifier/NameQualifierPresignPlugin.java#L68-L69